src/main/java/com/epam/jmp2017/model/json/ActionModel.java

```diff
@@ -7,9 +7,18 @@
 import com.epam.jmp2017.model.conditions.CompositeCondition;
 import com.epam.jmp2017.model.enums.ActionType;
 import com.epam.jmp2017.model.enums.Attribute;
-import com.epam.jmp2017.model.loaders.ConditionsLoader;
+import com.epam.jmp2017.util.loaders.ConditionsLoader;
+import org.springframework.beans.factory.annotation.Autowired;
 
 public class ActionModel {
+
+    private static ConditionsLoader conditionsLoader;
```

Comment:
Otherwise Gson goes in cyclic initializations and produce StackOverflowException